pkgs/tools/security/rnp/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, stdenv
+, asciidoctor
+, botan2
+, bzip2
+, cmake
+, fetchFromGitHub
+, json_c
+, pkg-config
+, zlib
+}:
+let
+  version = "0.15.0";
+in
+stdenv.mkDerivation rec {
+  pname = "rnp";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "rnpgp";
+    repo = "rnp";
+    rev = "v${version}";
+    sha256 = "1vpc37c3fb1lhfaywd0pzi42lw0vadav4xgdfkzlr27759hfmhxc";
+  };
+
+  buildInputs = [ zlib bzip2 json_c botan2 ];
+
+  cmakeFlags = [
+    "-DCMAKE_INSTALL_PREFIX=${placeholder "out"}"
+    "-DBUILD_SHARED_LIBS=on"
+    "-DBUILD_TESTING=off"
```

Comment:
Why do we not want to run tests?

Reply:
Running tests would involve fetching various things like `googletest` from the Internet, so it's not as straightforward as we'd like.

Reply:
We already have googletest packaged. I think we should try it to verify that the package works correctly.

Reply:
Thanks for the pointers.  I've now included `gtest` and enabled tests in the latest commit.